libraries/helpers.rb

```diff
@@ -73,24 +98,37 @@ def initialized?
       def secure_random
         r = SecureRandom.hex
         Chef::Log.debug "Generated password: #{r}"
+
         r
       end
 
-      def default_server_packages
+      def default_server_packages(version: nil, source: :os)
```

Comment:
We shouldn't be defaulting source type here. That changes the default behaviour right?

Reply:
It won't change the default because its always specified, but yes we shouldn't be setting a default here you're right.